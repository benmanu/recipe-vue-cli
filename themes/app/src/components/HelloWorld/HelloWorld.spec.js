import { shallowMount } from '@vue/test-utils';
import HelloWorld from './HelloWorld.vue';

describe('HelloWorld.vue', () => {
  it('renders props.msg when passed', () => {
    const msg = 'new message';
    const wrapper = shallowMount(HelloWorld, {
      propsData: { msg },
    });
    expect(wrapper.text()).toMatch(msg);
  });

  it('renders correctly', () => {
    const wrapper = shallowMount(HelloWorld);
    expect(wrapper).toMatchSnapshot();
  });
});
