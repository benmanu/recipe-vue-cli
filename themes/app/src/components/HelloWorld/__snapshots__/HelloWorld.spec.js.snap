// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HelloWorld.vue renders correctly 1`] = `
<div>
  <h1></h1>
</div>
`;
